src/braginskii_collisions.cxx

```diff
@@ -49,25 +46,26 @@ Collisions::Collisions(std::string name, Options& alloptions, Solver*) {
       options["diagnose"].doc("Output additional diagnostics?").withDefault<bool>(false);
 }
 
-/// Calculate transfer of momentum and energy between species1 and species2
+/// Calculate apply collision data for the two species.
 /// nu_12    normalised frequency
 ///
 /// Modifies
 ///   species1 and species2
 ///     - collision_frequency
-///     - momentum_source   if species1 or species2 velocity is set
-///     - energy_source     if species1 or species2 temperature is set
-///                         or velocity is set and frictional_heating
 ///
 /// Note: A* variables are used for atomic mass numbers;
 ///       mass* variables are species masses in kg
-void Collisions::collide(Options& species1, Options& species2, const Field3D& nu_12, BoutReal momentum_coefficient) {
+void BraginskiiCollisions::collide(Options& species1, Options& species2,
+                                   const Field3D& nu_12) {
   AUTO_TRACE();
 
-  add(species1["collision_frequency"], nu_12);                           // Total collision frequency
-  std::string coll_name = species1.name() + std::string("_") + species2.name() + std::string("_coll");
-  set(species1["collision_frequencies"][coll_name], nu_12);              // Collision frequency for individual reaction
-  set(collision_rates[species1.name()][species2.name()], nu_12);         // Individual collision frequency used for diagnostics
+  add(species1["collision_frequency"], nu_12); // Total collision frequency
+  std::string coll_name =
+      species1.name() + std::string("_") + species2.name() + std::string("_coll");
+  set(species1["collision_frequencies"][coll_name],
+      nu_12); // Collision frequency for individual reaction
```

Comment:
`clang-format` is a little annoying about comments at the end of lines -- can be nicer to move them to the line before:

```suggestion
  // Collision frequency for individual reaction
  set(species1["collision_frequencies"][coll_name], nu_12);
```